config/config.yaml

```diff
@@ -48,7 +48,7 @@ branch-protection:
             - pull-cert-manager-bazel
             - pull-cert-manager-deps
             - pull-cert-manager-chart
-            - pull-cert-manager-e2e-v1-15
+            - pull-cert-manager-e2e-v1-17
```

Comment:
Is this Kubernetes 1.15 > 1.17 ?

Reply:
Yep

Reply:
We set `always_run: true` on the v1.17 job as well as the v1.15 job last week, so all recently opened PRs will already have test results for this job 😄 (gradual 'rollover' as it were). Now that we're seeing the job passing we're ready to switch it over and make it the required one (as we don't need to run both on every pull request!)